specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/MigrateFromSmartDetections.json

```diff
@@ -0,0 +1,273 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-01-01-preview",
+    "title": "Azure Alerts Management Service Resource Provider",
+    "description": "APIs for managing a migration from Smart Detection to smart alerts."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection": {
+      "post": {
+        "tags": [
+          "MigrateFromSmartDetection"
+        ],
+        "description": "Starts an asynchronous migration process of Smart Detection to smart alerts in an Application Insights resource.",
+        "operationId": "MigrateFromSmartDetection_StartMigration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SmartDetectionMigrationRequest"
+            },
+            "description": "The parameters controlling the migration process."
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
>"202" [](start = 10, length = 5)

You are missing a 200 response please refer to [this](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#call-action-post-asynchronously) doc, additionally you must add: "x-ms-long-running-operation": true like mentioned [here](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation)

Reply:
I have added the x-ms-long-running-operation
however, why do I need 200? the post will not return 200, it will return in location the url for the get status (the get api in this swagger), and that one will return 200 . isn't that valid?

Reply:
Got it, your location is the one reporting the status, then no need for 200, what you have is ok.